vanilla-framework/vanilla-dashboard-theme
# Contextual menu

## Done

- Created markup and SCSS for the new contextual element pattern.

## QA

- Pull down the branch, and do ```./run build```
- View the example here and check against the designs http://0.0.0.0:8102/vanilla-dashboard-theme/examples/patterns/contextual-menu/contextual-menu/

## Details

Fixes #6 

## Screenshots

[if relevant, include a screenshot]
